front_end/src/components/landing_page/Landing.js

```diff
@@ -19,15 +19,7 @@ class Landing extends Component {
   render() {
     const { classes } = this.props;
 
-    return (
-      <div className="landing">
-        <Grid container spacing={24} justify="center">
-          <Grid item xs={12} lg={9}>
-            <SearchWidget />
-          </Grid>
-        </Grid>
-      </div>
-    );
+    return <div className="landing" />;
```

Comment:
dang, killed my baby

Reply:
LOL. It will be back in the landing page. Just had to put a blank white space to demonstrate the navbar and footer.